sys-process/iotop/iotop-0.6_p20230124.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..11} )
+PYTHON_REQ_USE="ncurses(+)"
+
+MY_COMMIT="a14256a3ff74eeee59493ac088561f1bafab85a7"
+inherit distutils-r1 linux-info
+
+DESCRIPTION="Top-like UI used to show which process is using the I/O"
+HOMEPAGE="http://guichaz.free.fr/iotop/"
+SRC_URI="https://repo.or.cz/iotop.git/snapshot/${MY_COMMIT}.tar.gz -> ${P}.tgz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~mips ~ppc ~ppc64 ~riscv ~sparc ~x86 ~amd64-linux ~x86-linux"
+
+RDEPEND="!sys-process/iotop-c"
+
+CONFIG_CHECK="~TASK_IO_ACCOUNTING ~TASK_DELAY_ACCT ~TASKSTATS ~VM_EVENT_COUNTERS"
+
+DOCS=( NEWS README THANKS )
+
+S="${WORKDIR}/${PN}-a14256a"
```

Comment:
Let's put this below SRC_URI (directly below) and also change it so it truncates the first N characters from MY_COMMIT, so you don't have to manually sync it in future.

Reply:
I knew I should have done this, but I don't know how to do it. Couldn't find any examples in ::gentoo either.

Reply:
hairu or whatever he calls himself now showed me how to do this.